trieste/acquisition/rule.py

```diff
@@ -294,3 +296,55 @@ def acquire(
         state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
 
         return point, state_
+
+
+class BatchAcquisitionRule(AcquisitionRule[None, SearchSpace]):
+    """ Implements a acquisition rule for a batch of query points """
+
+    def __init__(self, num_query_points: int,
+                 builder: BatchAcquisitionFunctionBuilder):
+        """
+        :param num_query_points: The number of points to acquire.
+        :param builder: The acquisition function builder to use.
+            :class:`BatchAcquisitionRule` will attempt to **maximise** the corresponding
+            acquisition function.
+            with tag `OBJECTIVE`.
+        """
+
+        if not num_query_points > 0:
+            raise ValueError(
+                f"Number of query points must be greater than 0, got {num_query_points}"
+            )
+
+        self._num_query_points = num_query_points
+        self._builder = builder
+
+    def _vectorize_batch_acquisition(self, acquisition_function):
+        return lambda at: acquisition_function(tf.reshape(at, at.shape[:-1].as_list() + [self._num_query_points, -1]))
```

Comment:
```suggestion
        return lambda at: acquisition_function(tf.reshape(at, at.shape[:-1] + [self._num_query_points, -1]))
```
you can add `TensorShape` and `list`